scorequaltrics_workflow_template.Rmd

```diff
@@ -48,12 +48,12 @@ fi
 * `id_column_name` = subject ID column name in Qualtrics survey; can be a regular expression
 
 ```{r}
-cred_file_location = '~/credentials.yaml.DEFAULT'
-sid_column_name = '(SID|ExternalDataReference|qid)'
-survey_name_filter = 'Freshman Project T.* Survey'
-sid_pattern = 'FP[0-9]{3}'
-exclude_sid = 'FP999' # subject IDs to exclude
-identifiableData = c('Name', 'EmailAddress', 'IPAddress') # exclude when printing duplicates
+cred_file_location = '~/credentials.yaml'
+sid_column_name = '(Login)'
+survey_name_filter = 'DEV Session.* Surveys'
+sid_pattern = 'DEV[0-9]{3}'
+exclude_sid = '(DEV999|test)' # subject IDs to exclude
+identifiableData = c('IPAddress', 'RecipientLastName', 'RecipientFirstName', 'RecipientEmail',	'ExternalReference', 'LocationLatitude',	'LocationLongitude') # exclude when printing duplicates
```

Comment:
Seems like there are some extra spaces or a tab character in the comma-separated list. Can you remove it?